.github/workflows/macos.yml

```diff
@@ -58,7 +58,7 @@ jobs:
     env:
       CC: ccache clang
       CXX: ccache clang++
-      FLAGS: -O3 -DNDEBUG -pipe -march=nehalem
+      FLAGS: -DNDEBUG -O3 -fno-math-errno -fstrict-aliasing -march=nehalem -flto=thin -pipe
```

Comment:
I am a little bit worried about `no-math-errno` in here.

Reply:
Clang's current vectorization documentation [recommends](https://llvm.org/docs/Vectorizers.html): _"To allow better optimization of C/C++ math library functions, use -fno-math-errno”._

I agree it adds risk; it's the math-equivalent to dropping asserts. I think in terms of likelihood, I don't recall ever hitting a math error, where as I have hit asserts during development. So from that perspective, I would think it's acceptable.  No worries though if you want to drop it for now (snip it after the merge?).